system-tests/projects/ts-proj-4-5/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "@vue/tsconfig/tsconfig.web.json"
```

Comment:
Any way to provide a more minimal tsconfig here? It's not clear what this config contains/why it's needed here.

Reply:
yes absolutely